05_syntax_lab/01.py

```diff
@@ -1,5 +1,14 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+print "This program will get10 numbers for user and return the biggset one"
+
+i=0
+number=0
+temp=0
+for i in range(0,10):
+    print "Please enter number:",
+    temp=int(raw_input())
+    if number < temp: number=temp
+    print
+
+print number
+
+

```

Comment:
Hi Dor, welcome to the exercises!

In the future, please keep the comment that came with the exercise files to make it easier to identify what the code is supposed to do.

* Usually there is no need to assign values to variable up-front. Please check which ones can you get rid of
* If the user provides 10 negative numbers, the result will be 0. Can you fix that please?
* See how can you merge lines 7 and 8 into one statement.
* Why do you need line 10 - the single print statement?